internal/core/data/v2/controller/http/event.go

```diff
@@ -416,3 +416,38 @@ func (ec *EventController) EventsByTimeRange(w http.ResponseWriter, r *http.Requ
 	utils.WriteHttpHeader(w, ctx, statusCode)
 	pkg.Encode(response, w, lc)
 }
+
+func (ec *EventController) AllReadings(w http.ResponseWriter, r *http.Request) {
```

Comment:
Move to `ReadingController` in `reading.go`

Reply:
Moved.